bin/compile

```diff
@@ -1,25 +1,32 @@
 #!/usr/bin/env bash
 
 indent() {
-    sed -u 's/^/      /'
+    sed -u 's/^/       /'
```

Comment:
I increased the indentation by one space as I noticed the indentation didn't actually line up as show below:
```
-----> Multi-procfile app detected
      Copied cmd/heroku/Procfile as Procfile successfully
-----> Discovering process types
       Procfile declares types -> stanbot
-----> Compressing...
       Done: 22M
-----> Launching...
```

Reply:
ha! Thanks for catching this. It's trivial, yet really annoying if you're paying attention to the output. :)